### ISSUE TITLE
There must be an empty line between `# coding` comment and module body

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#358]
# Rule request

## Thesis

We need to enforce consistent line breaking rule around `coding magic comment`. Currently it is possible to write:

```python
# -*- coding: utf-8 -*-
some_x = 1
```

But, it should be only allowed to write:

```python
# -*- coding: utf-8 -*-

some_x = 1
```

We need to check that if there's a comment on top of the module the next body element should come after a single new empty line.

## Reasoning

This is stylistic feature to enforce consistency.
